.github/workflows/cancel.yml

```diff
@@ -0,0 +1,15 @@
+name: Cancel
+on: [push]
```

Comment:
push only runs if you have the rights on the repository, not if you make a PR from a fork. How can we ensure to run this on fork PRs updates too?

Reply:
I changed it to trigger only on `pull_request` events, which should run on forked repos as well: https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-events-for-forked-repositories

Most of the current workflows are already triggered on `pull_request` and some also on `push` to `master`. This way it won't cancel those triggered via `push` to `master` and maybe that's better.